packages/aws-cdk/lib/api/hotswap/evaluate-cloudformation-template.ts

```diff
@@ -50,6 +50,11 @@ export class EvaluateCloudFormationTemplate {
     return stackResources.find(sr => sr.LogicalResourceId === logicalId)?.PhysicalResourceId;
   }
 
+  public async findLogicallNameFor(physicalId: string): Promise<string | undefined> {
```

Comment:
Let's be more precise in the names here:

```suggestion
  public async findLogicalIdForPhysicalName(physicalName: string): Promise<string | undefined> {